logstash-mixin-aws.gemspec

```diff
@@ -16,7 +16,7 @@ Gem::Specification.new do |s|
   s.test_files = s.files.grep(%r{^(test|spec|features)/})
 
   # Gem dependencies
-  s.add_runtime_dependency "logstash-core", '>= 1.4.0', '< 2.0.0'
+  s.add_runtime_dependency "logstash-core", ">= 2.0.0", "< 3.0.0"
```

Comment:
Well, I see few reason for this.

1. There is no real benefits of using single quote..
2. We have a mix and match of single quote / double quote, I though we had decided to use double quotes ;)
3. And it was a bit easier on my side for the regular expression

  content.gsub!(/add_runtime_dependency ("|')logstash-core("|').+/, 'add_runtime_dependency "logstash-core", ">= 2.0.0", "< 3.0.0"')